test/e2e/apps/replica_set.go

```diff
@@ -495,3 +502,48 @@ func testRSLifeCycle(f *framework.Framework) {
 	framework.ExpectEqual(*(rs.Spec.Replicas), rsPatchReplicas, "replicaset should have 3 replicas")
 	framework.ExpectEqual(rs.Spec.Template.Spec.Containers[0].Image, rsPatchImage, "replicaset not using rsPatchImage. Is using %v", rs.Spec.Template.Spec.Containers[0].Image)
 }
+
+// List and DeleteCollection operations
+func listRSDeleteCollection(f *framework.Framework) {
+
+	ns := f.Namespace.Name
+	c := f.ClientSet
+	rsClient := f.ClientSet.AppsV1().ReplicaSets(ns)
+	one := int64(1)
+	rsName := "test-rs"
+	replicas := int32(3)
+	e2eValue := rand.String(5)
+
+	// Define ReplicaSet Labels
+	rsPodLabels := map[string]string{
+		"name": "sample-pod",
+		"pod":  WebserverImageName,
+		"e2e":  e2eValue,
+	}
+
+	ginkgo.By("Create a ReplicaSet")
+	rs := newRS(rsName, replicas, rsPodLabels, WebserverImageName, WebserverImage, nil)
+	_, err := rsClient.Create(context.TODO(), rs, metav1.CreateOptions{})
+	framework.ExpectNoError(err)
+
+	ginkgo.By("Verify that the required pods have come up")
+	err = e2epod.VerifyPodsRunning(c, ns, "sample-pod", false, replicas)
+	framework.ExpectNoError(err, "Failed to create pods: %s", err)
```

Comment:
nit: Not pod creation but pods did not come up to Running State

Reply:
Good point. The test is following a similar pattern as the Conformance tests for other resources.
I will keep this in mind for the next test of this kind.